### ISSUE TITLE
Bittrex --> 0.0 amount for BCH (Bitcoin cash); cannot fetch amounts

### ISSUE THREAD

**Issue by nmiculinic** [ccxt/ccxt#475]
I cannot fetch BCH data for bittrex:

```
        exchange = ... (setup)
        util.pp.pprint(
            await exchange.fetch_total_balance()
        )
```

I normally get BTC>0, and other currencies, yet BCH remains zero despite having funds on the bittrex.

Version: 1.10.2

**Comment by nmiculinic** [comment]
I've checked all >0 tokens, nothing resembles BCH with positive amount.

**Comment by kroitor** [comment]
@nmiculinic thx for reporting this, I'll check it and will fix it very soon.

**Comment by kroitor** [comment]
@nmiculinic can you post verbose output (set `.verbose = True` on the exchange instance before calling the fetch_total_balance()) ?

**Comment by kroitor** [comment]
@nmiculinic never, mind, I figured it out, will fit it right away.

**Comment by kroitor** [comment]
@nmiculinic this issue was fixed in 1.10.4. Thx!
